.github/workflows/main.yml

```diff
@@ -1027,7 +1033,7 @@ jobs:
           - "wasmtime-environ"
           - "pulley-interpreter --all-features"
     needs: determine
-    if: needs.determine.outputs.run-full && github.repository == 'bytecodealliance/wasmtime'
+    if: (needs.determine.outputs.run-full || needs.determine.outputs.test-miri) && github.repository == 'bytecodealliance/wasmtime'
```

Comment:
This is ok to change to just `test-miri` since `run-full` always implies that `test-miri` is `true` above (most of the other jobs don't check both, they only check the one)